socialhome/streams/views.py

```diff
@@ -1,35 +1,28 @@
 from braces.views import LoginRequiredMixin
 from django.shortcuts import get_object_or_404
-from django.utils.translation import gettext
 from django.views.generic import ListView
-from django.urls import reverse
 
 from socialhome.content.models import Content, Tag
 
 
-class BaseStreamView(ListView):
-    model = Content
-    ordering = "-created"
-    paginate_by = 30
-    stream_name = ""
+class VueStreamMixin:
```

Comment:
Actually, I'm not sure why we're doing this whole mixin? The API is close, juggling the stream views at this point is IMHO unnecessary as in a day or two you can start pulling content out of the API. Then we can remove the whole JSON context from the view (though not the template override).